companies/tradedoubler.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "tradedoubler",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "ads"
+    ],
+    "name": "Tradedoubler AB",
+    "runs": [
+        "Tradedoubler GmbH",
+        "Adnologies",
+        "Interactive Marketing Works Ltd"
+    ],
+    "address": "Birger Jarlsgatan 57A\n113 56 - Stockholm\nSweden\n",
+    "phone": "+46 8 405 08 00",
+    "email": "privacy@tradedoubler.com",
+    "sources": [
+        "http://performancein.com/news/2015/01/26/tradedoubler-completes-deal-adnologies/",
+        "https://en.wikipedia.org/wiki/Tradedoubler",
+        "https://www.tradedoubler.com/en/privacy-policy/",
+        "https://www.tradedoubler.com/en/about-us/"
+    ],
+    "required-elements": [
```

Comment:
As the test is helpfully telling you, if you specify required elements, you need a `name` element.

Reply:
```suggestion
    "required-elements": [
       {
            "desc": "Name",
            "type": "name"
       },
```